### PR TITLE
slcli server create-options dal13 Error

### PR DESCRIPTION
**Pull request by caberos**
https://github.com/softlayer/softlayer-python/issues/1525

### CODE REVIEW

File: SoftLayer/managers/account.py [hunk 1]
```diff
@@ -277,7 +277,7 @@ def get_account_all_billing_orders(self, limit=100, mask=None):
         return self.client.call('Billing_Order', 'getAllObjects',
                                 limit=limit, mask=mask)
 
-    def get_routers(self, mask=None, location=None):
+    def get_routers(self, location=None, mask=None):
```
Comment (FernandoOjeda) [inline SoftLayer/managers/account.py:280]: I suggest you to fix it direct in the cli hardware create-options module, like this:
```python
routers = account_manager.get_routers(location=location)
```

Comment (allmightyspiff) [reply SoftLayer/managers/account.py:280]: agreed, being specific about parameters can help avoid bugs like this in the future.

**Review by allmightyspiff — CHANGES_REQUESTED** [review]

**Review by allmightyspiff — CHANGES_REQUESTED** [review]
I think you accidently reverted your change to `SoftLayer/managers/account.py`, can you put the function back to `def get_routers(self, location=None, mask=None):` and this will be good to go. Sorry if there was any confusion about that.

**Review by allmightyspiff — APPROVED** [review]
Looks good. thanks.

**Pull request merged**